api/hearings/data/hearingRequests.mock.data.ts

```diff
@@ -104,7 +103,6 @@ export const HEARING_REQUEST_RESULTS: HearingRequestMainModel[] = [
         partyID: 'P2',
         partyName: 'DWP',
         partyType: PartyType.ORG,
-        partyChannel: 'byVideo',
```

Comment:
in this mock data, after the partyChannel: 'inPerson', is removed here, could this be added to the the individualDetails.preferredChannel.